yt/opensource-packages/ytsaurus/Dockerfile

```diff
@@ -0,0 +1,121 @@
+ARG SOURCE_ROOT="/ytsaurus"
+ARG BUILD_ROOT="/build"
+
+ARG PROTOC_VERSION="3.20.1"
+
+FROM ubuntu:focal as build-python-packages
+
+ARG SOURCE_ROOT
+ENV SOURCE_ROOT $SOURCE_ROOT
+
+ARG BUILD_ROOT
+ENV BUILD_ROOT $BUILD_ROOT
+
+RUN apt-get update && apt-get install -y software-properties-common
+RUN add-apt-repository ppa:deadsnakes/ppa
+
+RUN apt-get install -y --no-install-recommends \
+      curl \
+      unzip \
+      python3 \
+      python3-pip \
+      python3.11 \
+      python3.11-venv \
+    && rm -rf /var/lib/apt/lists/*
+
+ARG PROTOC_VERSION
+ENV PROTOC_VERSION $PROTOC_VERSION
+
+RUN curl -sL -o protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip \
+    && unzip protoc.zip -d /usr/local \
+    && rm protoc.zip
+
+RUN mkdir -p ${SOURCE_ROOT}
+COPY ./yt/python ${SOURCE_ROOT}/yt/python
+COPY ./yt/yt/python ${SOURCE_ROOT}/yt/yt/python
+COPY ./certs/cacert.pem ${SOURCE_ROOT}/certs/cacert.pem
+
+RUN mkdir -p ${BUILD_ROOT}
+COPY ./artefacts/libyson_lib.so ${BUILD_ROOT}/yt/yt/python/yson_shared/libyson_lib.so
+COPY ./artefacts/libdriver_lib.so ${BUILD_ROOT}/yt/yt/python/driver/native_shared/libdriver_lib.so
+COPY ./artefacts/libdriver_rpc_lib.so ${BUILD_ROOT}/yt/yt/python/driver/rpc_shared/libdriver_rpc_lib.so
+
+RUN python3.11 -m venv package-env
+RUN . package-env/bin/activate; \
+    ${SOURCE_ROOT}/yt/python/packages/build_ytsaurus_packages.sh --ytsaurus-source-path ${SOURCE_ROOT} --ytsaurus-build-path ${BUILD_ROOT} --apply-auditwheel
+
+FROM ubuntu:focal as build-ytsaurus
+
+USER root
+
+WORKDIR /tmp
+
+# Main binary.
+COPY ./ytserver-all /usr/bin/ytserver-all
+
+# If this list changes, also update yt_nightly/Dockerfile
+RUN ln -s /usr/bin/ytserver-all /usr/bin/ytserver-master && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-master-cache && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-clock && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-discovery && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-node && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-job-proxy && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-exec && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-proxy && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-http-proxy && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-tools && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-scheduler && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-controller-agent && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-query-tracker && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-queue-agent && \
+    ln -s /usr/bin/ytserver-all /usr/bin/ytserver-tcp-proxy
+
+COPY ./init_queue_agent_state.py /usr/bin/init_queue_agent_state
+COPY ./init_operations_archive.py /usr/bin/init_operations_archive
```

Comment:
Why is there no `init_query_tracker_state` here?

Reply:
I didn't add it because it isn't present in the main image either: https://github.com/ytsaurus/ytsaurus/blob/main/yt/docker/ytsaurus/Dockerfile#L29. 
I gathered that's because we have a separate query tracker image.

Surprisingly, we do create a `ytserver-query-tracker` symlink, both in the main image and in this PR. Seems like something redundant.